### DISCUSSION TITLE
话说各位在GitHub上有多少源仓库

### DISCUSSION THREAD

**Discussion by Myitian** [share121/inter-knot#1661] [其他]
![](https://github.com/user-attachments/assets/6021c4e6-c5db-4edb-ad5c-f388c50d0a3b)
就是在仓库界面选 Sources 看到的那种，这个不包括 Fork 出来的

**Comment by inkluv** [comment]
3个好像（都是没用的），有一个是初中时候fork了一个sansfight（

**Comment by Buer-Nahida** [comment]
10个，但实际就4个有用的，3个neovim插件一个NixOS flake

**Comment by capacitor1** [comment]
一堆但大多是fork的。
自己写的那几个都是废物
